packages/contracts-bedrock/src/L2/OperatorFeeVault.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Contracts
+import { FeeVault } from "src/L2/FeeVault.sol";
+
+// Libraries
+import { Types } from "src/libraries/Types.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+
+// Interfaces
+import { ISemver } from "interfaces/universal/ISemver.sol";
+
+/// @custom:proxied true
```

Comment:
Perfect, this simplifies things. Thank you for making this change. Now we can reduce diff with the config, where setting the value is fished thru `op-deployer` and other places